@balazsorban44 
getting the following error, even though I don't use mysql
```
../../node_modules/.pnpm/@auth+drizzle-adapter@0.0.0-manual.acabcfb4_nodemailer@6.9.4/node_modules/@auth/drizzle-adapter/index.js:18:0
Module not found: Package path ./mysql-core/index.js is not exported from package /Users/ssardorf/dev/12s12m/verk/node_modules/.pnpm/node_modules/drizzle-orm (see exports field in /Users/ssardorf/dev/12s12m/verk/node_modules/.pnpm/node_modules/drizzle-orm/package.json)

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./lib/auth.tsx
./app/api/auth/[...nextauth]/route.tsx
```

--- a/packages/adapter-drizzle/src/index.ts
+++ b/packages/adapter-drizzle/src/index.ts
@@ -16,12 +16,9 @@
  * @module @auth/drizzle-adapter
  */
 
-import { MySqlDatabase, MySqlTableFn } from "drizzle-orm/mysql-core/index.js"
-import { PgDatabase, PgTableFn } from "drizzle-orm/pg-core/index.js"
-import {
-  BaseSQLiteDatabase,
-  SQLiteTableFn,
-} from "drizzle-orm/sqlite-core/index.js"
+import { MySqlDatabase, MySqlTableFn } from "drizzle-orm/mysql-core"
+import { PgDatabase, PgTableFn } from "drizzle-orm/pg-core"
+import { BaseSQLiteDatabase, SQLiteTableFn } from "drizzle-orm/sqlite-core"
 import { mySqlDrizzleAdapter } from "./lib/mysql.js"
 import { pgDrizzleAdapter } from "./lib/pg.js"
 import { SQLiteDrizzleAdapter } from "./lib/sqlite.js"

